children/schema.py

```diff
@@ -295,12 +296,40 @@ class GuardianInput(graphene.InputObjectType):
     has_accepted_marketing = graphene.Boolean()
 
 
-class ChildInput(graphene.InputObjectType):
+class BaseChildInput:
+    """
+    A base class for input fields shared by all Child inputs
+    """
+
     name = graphene.String()
-    birthyear = graphene.Int(required=True)
-    postal_code = graphene.String(required=True)
     relationship = RelationshipInput()
     languages_spoken_at_home = graphene.List(graphene.NonNull(graphene.ID))
+    notes = graphene.String()
```

Comment:
I would move the "notes management" to a totally new Query-Mutation -pair, that would have only this 1 field as an input (and output).

So, I would create a 
1. `ChildAttendanceHistoryQuery(childId: str)`
2. ChildAttendanceHistorInput
3. `ChildAttendanceHistoryMutation(input: ChildAttendanceHistorInput)`

The `ChildAttendanceHistorInput` would have structure something like
```
input: {
    childId: str,
    content: str
}
```
The field name coud be content/history/text/notes or something like that.

Reply:
When thinking the UI, I just don't think that the attendance history notes fits well in the `ChildEditForm`, because it's more like personal contact information and the very basic information. Another thing is that when we are mutating the history notes, I would not want it to have all the other critical child information as a must have in the input. 

I think this new feature is a totally unrelated / new feature that should not be mixed with any of the existing features.

So instead of adding the field here in the EditChildForm 
![image](https://github.com/City-of-Helsinki/kukkuu/assets/389204/c95e95f4-8ba5-4f80-8300-4be227b5f768)
I would add the form in a new passive card in the child's events listing, somehow like this:
![image](https://github.com/City-of-Helsinki/kukkuu/assets/389204/1ea60b91-93c1-47e7-bac1-4c52a1d805dc)
When the "Muokkaa historiaa" is clicked, a text area would replace the text content.

Reply:
That looks like a better choice UI speaking hmm